<launch>

    <arg name="pc_topic" default="/mock_point_cloud" />
    <arg name="pc_bag" default="$(find shuguru)/data/simulation/shelf_2x2_boxes.bag" />
    <!-- "/current_point_cloud" for real robot-->

    <include file="$(find rosbridge_server)/launch/rosbridge_websocket.launch" />
    <include file="$(find applications)/launch/navigation_sim.launch" />
    <include file="$(find fetch_api)/launch/ar_desktop.launch">
        <arg name="cam_image_topic" value="$(arg pc_topic)" />
    </include>
    <include file="$(find fetch_api)/launch/move_group.launch" />
    <!--<node pkg="shuguru" type="publish_saved_cloud.py" name="fetch_publish_saved_cloud" output="screen" args="$(arg pc_bag)" />-->
    <node pkg="applications" type="gripper_im_server.py" name="gripper_im_server" output="screen" />
    <node pkg="shuguru" type="command_server.py" name="command_server" output="screen"/>
    <node pkg="shuguru" type="manipulation_server.py" name="manipulation_server" output="screen"/>
    <node pkg="shuguru" type="state_machine.py" name="state_machine" output="screen"/>
    <!--<node pkg="tf" type="static_transform_publisher" name="shelf_broadcaster" args="2.0 0.5 0.0 0.0 0.0 0.0 1 map shelf 100" />-->

</launch>
